build-system/tasks/visual-diff.js

```diff
@@ -17,13 +17,23 @@
 
 const argv = require('minimist')(process.argv.slice(2));
 const execOrDie = require('../exec').execOrDie;
+const getStdout = require('../exec').getStdout;
 const gulp = require('gulp-help')(require('gulp'));
 
 
 /**
  * Simple wrapper around the ruby based visual diff tests.
  */
 function visualDiff() {
+  if (!argv.master) {
+    const userName = getStdout(
+        'git log -1 --pretty=format:"%ae"').trim();
```

Comment:
Still works for cherry-picks? Top commit might not be from PR author e.g. https://github.com/ampproject/amphtml/pull/11529/commits

Reply:
Excellent question, I'm glad you asked it. The answer is yes, it will work.

This is because we do push builds for release branches. (...and not PR builds, as we established earlier this week!) And when we do a push build from Travis, we pass in `--master` to `gulp visual-diff`. (I should probably have named that arg `--push` instead of `--master`, but you get the point.)

Let me know if this sounds convincing.

Reply:
SGTM, can you add a brief comment?

Reply:
Done in the last commit via new comment at the top of the function.